src/index.css

```diff
@@ -18,7 +18,7 @@ body {
   min-height: 100vh;
   text-rendering: optimizeSpeed;
   line-height: 1.5;
-  background: white!important;
+  background: white !important;
```

Comment:
[Pebble] Do we need "important" in these two places?

Reply:
Thank you, I will remove those that are not necessary